nixos/tests/nextcloud/with-postgresql-and-redis.nix

```diff
@@ -46,7 +46,7 @@ in {
       systemd.services.redis = {
         preStart = ''
           mkdir -p /var/run/redis
-          chown ${config.services.redis.user}:${config.services.nginx.group} /var/run/redis
+          chown redis:${config.services.nginx.group} /var/run/redis
```

Comment:
If we want the test to keep using the file socket (and not link-local networking instead, we should instead set `unixsocketperm` in the test to something more permissive.

Reply:
I'm honestly not sure what you mean,  :)

Reply:
do you mind writing what you think would be appropriate?

Reply:
These lines seem to exist only to get php to be able to connect to redis.

Further down, `configureRedis = pkgs.writeScript "configure-redis"` runs `nextcloud-occ` and configures nextcloud to connect to redis at `/var/run/redis/redis.sock`.

This `preStart` tries to extend the existing `redis` systemd service to obtain a less restrictive `/run/redis` folder (which contains the unix socket file). 
However, `/run/redis` and its permissions are managed by systemd's `RuntimeDirectory` nowadays (and `/var/run/ is a symlink to `/run`), so this doesn't have the desired effect.

I'd suggest to drop the `preStart`, `services.redis.unixSocket` and `services.redis.extraConfig`, and configure nextcloud to connect to `localhost:6379`.

Reply:
makes more sense, i don't dabble much with systemd, it's next on my plate. Thanks!

Reply:
Do you feel confident to update the PR with this suggestions?

Reply:
I do, just slammed in my personal life with obligations (friends visiting from out of town)

Reply:
No worries :-)